test/api_test.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+import json
+
+import configparser
+
+from glusterapi import Client
+
+
+class TestPeerInfo(object):
+	peerID = ''
+	host = ''
+	node_list = []
+
+	@classmethod
+	def set_host(cls, host):
+		cls.host = host
+
+	@classmethod
+	def set_peer_id(cls, peerid):
+		cls.peerID = peerid
+
+	@classmethod
+	def set_node_list(cls, node):
+		cls.node_list.append(node)
+
+	@property
+	def get_peer_id(self):
+		return self.peerID
+
+	@property
+	def get_host(self):
+		return self.host
+
+	@property
+	def get_node_list(self):
+		return self.node_list
+
+
+Peer = TestPeerInfo()
+
+
+@pytest.fixture(scope='module')
+def gd2client():
+    config = json.loads(open('config.json').read())
+    endpoint = config["glusterd2"]["endpoint"]
+    user = config["glusterd2"]["user"]
+    secret = config["glusterd2"]["secret"]
+    verify = config["glusterd2"]["verify"]
+    host = config["peer"]["host1"]["hostname"]
+    node = config["peer"]["host2"]["hostname"]
+    # store host in TestPeerInfo(controller node)
+    Peer.set_host(host)
+    Peer.set_node_list(node)
+
+    return Client(endpoint=endpoint, user=user, secret=secret, verify=verify)
+
+
+def test_peer_add(gd2client):
+    status, resp = gd2client.peer_add(host=Peer.get_host)
+    # store  peerID in class variable
+    Peer.set_peer_id(resp['id'])
+    assert status == 201
+
+
+def test_peer_status(gd2client):
+    status, resp = gd2client.peer_status()
+    # assert resp[0]['id'] == Peer.get_peer_id)
+    assert status == 200
```

Comment:
asserting with status code won't help us, we need to check something like the length of a response.